appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/view-and-review.controller.js

```diff
@@ -112,6 +112,12 @@ export const postReviewInterestedPartyComment = async (request, response) => {
 			});
 		}
 
+		if (status === 'valid_requires_redaction') {
```

Comment:
Just checking, should this come from a constant?

Reply:
Yeah I should probably constanify it. I was being sloppy because this status is just for the UI, it never goes to the db